app/src/main/java/org/mozilla/fenix/utils/DeleteAndQuit.kt

```diff
@@ -17,7 +16,7 @@ import org.mozilla.fenix.settings.DefaultDeleteBrowsingDataController
  */
 fun Context.deleteAndQuit(coroutineScope: CoroutineScope) {
```

Comment:
We should follow up later to make this an function that takes in activity rather than using context, since it gets cast anyways.